docs/getting_started.md

```diff
@@ -0,0 +1,132 @@
+<p align="center">
+<img alt="BankAxept_Logo.svg" src="../assets/images/bankaxept_logo.svg" width="300"/>
+</p>
+
+```
+Welcome to the Getting Started guide for the BankAxept 
+Epayment Platform (EPP).
+
+This text is intended to give both clear descriptions and 
+guidance for integrators and consumers of EPP. It includes an
+overview of all critical features as well as some general
+hints of how to utilize this platform.
+```
+
+# Introduction
+EPaymentPlatform (EPP) is a Payment API for integrators (Integrator) utilizing BankAxept online payments. 
+It is based on a core principle of asynchronous exchange of information where the transactions created can be identified using an EPP defined `PaymentId` and an Integrator defined `MessageId`. 
+Traditional transaction operations can thereafter be performed as seen according to the `Operations` section.
+
+## Setting up your EPP integration
+
+In order to set up your EPP integration and start requesting payments the following operations need to 
```

Comment:
Sorry, but this could use a comma after "payments" 😅

Reply:
Agreed, resolved! :D